src/ProtectedProxies/ProtectedUUPSUpgradeable.sol

```diff
@@ -25,6 +26,26 @@ abstract contract ProtectedUUPSUpgradeable is UUPSUpgradeable {
         }
     }
 
+    /**
+     * Returns an address from an arbitrary slot.
+     * @param slot to read an address from
+     */
+    function _getAddress(bytes32 slot) internal view returns (address addr) {
```

Comment:
If this is the only place you use `sload` then name the function _getSubImplementation instead of arbitrary read